### ISSUE TITLE
chapter 6 cat dog classifier problem

### ISSUE THREAD

**Issue by dandengAI** [chenyuntc/pytorch-book#58]
when I use Resnet34 model I run 40 epoches I got very good result!!
but when I switch to AlexNet model,  the loss value stuck at 0.6945 the accuracy stucked at 0.5

**Comment by chenyuntc** [comment]
You are Right. Because  AlexNet requires strict initialization, learning rate and many other things. It's not a  good implementation here.
I will replace it by squeezenet in the next version. Sorry for troubling.
